app/views/companies/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= @company.symbol %><br>
+<%= @company.name %><br>
```

Comment:
HTML tag should be closed at some point. For example, you can do `<tag></tag>` or `<tag />`.

In your case:

```suggestion
<%= @company.name %><br />
```